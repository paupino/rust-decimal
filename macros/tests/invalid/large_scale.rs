use rust_decimal_macros::dec;

fn main() {
    let _ = dec!(0.00000_00000_00000_00000_00000_0001);
}
