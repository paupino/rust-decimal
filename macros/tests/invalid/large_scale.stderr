error: proc macro panicked
 --> tests/invalid/large_scale.rs:4:13
  |
4 |     let _ = dec!(0.00000_00000_00000_00000_00000_0001);
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Number has a high precision that can not be represented.
